### DISCUSSION TITLE
[Question]: Thumnail object detection missing

### DISCUSSION THREAD

**Discussion by HolgerM1** [blakeblackshear/frigate#10182] [Ask A Question] [answered]
### What is your question:

Hi,
for meanwhile a week or so, the thumbnails do not work anymore. The entity is missing?! I did not change anything aside from updating the HA core and OS. Frigate seems to be working normally. I can see all detections within frigate.
![Screenshot 2024-02-24 145149](https://github.com/blakeblackshear/frigate/assets/114694650/af69a620-c2d0-48a1-b057-c1d24ee85b56)

Anybody having the same issue?

Thanks!

**Comment by NickM-27** [comment, marked as answer]
The camera.object entites were removed in favor of image.object entities

**Reply by HolgerM1** [reply]
Sorry for asking again, but could you give me a hint how to solve this?

Edit: Found out myself. I just hat to change the entites form "camera.garage_person" to "image.camera_person"
